Oracle/cdm_transform_tests.sql

```diff
@@ -528,6 +528,38 @@ order by 2
 ;
 */
 
+
+/*  Verify that LAB_RESULT_CM contains data associated with a least half of the 
```

Comment:
I'd like to see a test that the average lab value is close to clinical norms (which we should be able to find in wikipedia or the like).